### PR TITLE
Start Guide for RAPIDS on AWS EMR 6.2

### PR DESCRIPTION
**Pull request by mgzhao**

### CODE REVIEW

File: docs/get-started/getting-started-aws-emr.md
Comment (jlowe) [inline docs/get-started/getting-started-aws-emr.md:1]:
```suggestion
# Get Started with RAPIDS on AWS EMR

```

File: docs/get-started/getting-started-aws-emr.md
Comment (jlowe) [inline docs/get-started/getting-started-aws-emr.md:3]:
```suggestion
This is a getting started guide for the RAPIDS Accelerator for Apache Spark on AWS EMR. At the end of this guide, the user will be able to run a sample Apache Spark application that runs on NVIDIA GPUs on AWS EMR.

```

File: docs/get-started/getting-started-aws-emr.md
Comment (jlowe) [inline docs/get-started/getting-started-aws-emr.md:5]:
```suggestion
The current EMR 6.2.0 release supports Spark version 3.0.1 and RAPIDS Accelerator version 0.2.0. For more details of supported applications, please see the [EMR release notes](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-release-6x.html).  

```

File: docs/get-started/getting-started-aws-emr.md
Comment (jlowe) [inline docs/get-started/getting-started-aws-emr.md:111]: This link brings the user to the main page for the RAPIDS Accelerator source code, not to more examples.  @sameerz @krajendrannv do you have a more appropriate link you'd like to see here?

File: docs/get-started/getting-started-aws-emr.md
Comment (jlowe) [inline docs/get-started/getting-started-aws-emr.md:44]: Elsewhere in the `docs/` directory we're using `img` as the directory name to hold images.  It would be nice to keep this consistent.  Also RAPIDS should be all caps in the filenames.  Interesting it's the _only_ part of the image filenames that is not capitalized. :smile:

**Review by jlowe — CHANGES_REQUESTED** [review]
In addition to the specific comments I have some general ones:
- It would be nice to have all the files specific to AWS EMR be placed in a directory that is specific to EMR (e.g.: `docs/get-started/aws-emr`).  Otherwise the `docs/get-started` directory is going to get even more cluttered as more platforms are added.
- The `docs/FAQ.md` should be updated to add a link to the new `getting-started-aws-emr.md` document.

File: docs/get-started/getting-started-aws-emr.md
Comment (tgravescs) [inline docs/get-started/getting-started-aws-emr.md:44]: some of hte configs in EMR guide are not right or should be adjusted for cluster setup.

Comment (krajendrannv) [reply docs/get-started/getting-started-aws-emr.md:44]: @mgzhao Tom pointed out that the config.json didn't have the right settings. I suggest that we provide a sample config.json here. For example; in this cluster you chose g4dn.2xlarge, which has 8 vCPU cores. The right config would be use all the cores, we need to spark.executor.cores=8, spark.task.resource.gpu.amount=0.125.
Here is the sample config for this cluster:
   {
      "Classification":"spark-defaults",
      "Properties":{
     	"spark.plugins":"com.nvidia.spark.SQLPlugin", 
     	"spark.sql.sources.useV1SourceList":"",
     	"spark.executor.extraJavaOptions":"-Dai.rapids.cudf.prefer-pinned=true",
     	"spark.executor.resource.gpu.discoveryScript":"/usr/lib/spark/scripts/gpu/getGpusResources.sh",      	
     	"spark.submit.pyFiles":"/usr/lib/spark/jars/xgboost4j-spark_3.0-1.0.0-0.2.0.jar",
     	"spark.executor.extraLibraryPath":"/usr/local/cuda/targets/x86_64-linux/lib:/usr/local/cuda/extras/CUPTI/lib64:/usr/local/cuda/compat/lib:/usr/local/cuda/lib:/usr/local/cuda/lib64:/usr/lib/hadoop/lib/native:/usr/lib/hadoop-lzo/lib/native:/docker/usr/lib/hadoop/lib/native:/docker/usr/lib/hadoop-lzo/lib/native",
     	"spark.rapids.sql.concurrentGpuTasks":"4",
     	"spark.executor.resource.gpu.amount":"1",
     	"spark.executor.cores":"8",
     	"spark.task.cpus ":"1",
     	"spark.task.resource.gpu.amount":"0.125",
     	"spark.rapids.memory.pinnedPool.size":"2G",
     	"spark.executor.memoryOverhead":"2G",
     	"spark.locality.wait":"0s",
     	"spark.sql.shuffle.partitions":"200",
     	"spark.sql.files.maxPartitionBytes":"256m",
      "spark.sql.adaptive.enabled":"false"
  	}

File: docs/get-started/getting-started-aws-emr.md
Comment (krajendrannv) [reply docs/get-started/getting-started-aws-emr.md:111]: yes @jlowe agreed. We need to point to the right ETL notebook and XGBoost Notebook

**Review by jlowe — DISMISSED** [review]
Minor extra suggestion, but I'm going to go ahead and approve this now since I'll be on vacation for a bit.  I don't want to totally block this for the 0.3 release.  However it would be good to get a review from @sameerz or @krajendrannv before committing this.

File: docs/get-started/getting-started-aws-emr.md
Comment (sameerz) [reply docs/get-started/getting-started-aws-emr.md]: I updated the FAQ.md and created subdirectories in demos and img for different CSPs.  I also updated the file names.  I believe Kong had already updated the config settings in the json.

File: docs/get-started/getting-started-aws-emr.md
Comment (sameerz) [reply docs/get-started/getting-started-aws-emr.md]: Updated the text.

**Comment by sameerz** [comment]
I updated the content, and modified a few related files (FAQ.md, the databricks and gcp dataproc getting started guides) as the img locations were also cleaned up.  Please review.

File: docs/get-started/getting-started-aws-emr.md
Comment (jlowe) [inline docs/get-started/getting-started-aws-emr.md:46]:
```suggestion
The `my-configurations.json` installs the spark-rapids plugin on your cluster, configures YARN to use

```

File: docs/get-started/getting-started-aws-emr.md
Comment (jlowe) [inline docs/get-started/getting-started-aws-emr.md:47]:
```suggestion
GPUs, configures Spark to use RAPIDS, and configures the YARN capacity scheduler.  An example JSON

```

**Review by jlowe — CHANGES_REQUESTED** [review]

**Comment by jlowe** [comment]
Looks good but there are some conflicts that need to be resolved.

File: docs/get-started/getting-started-aws-emr.md
Comment (tgravescs) [inline docs/get-started/getting-started-aws-emr.md:128]: not needed anymore

File: docs/get-started/getting-started-aws-emr.md
Comment (tgravescs) [inline docs/get-started/getting-started-aws-emr.md:132]: 4 might be a bit aggressive, 2 may be a safer default

File: docs/get-started/getting-started-aws-emr.md
Comment (tgravescs) [inline docs/get-started/getting-started-aws-emr.md:241]: extra newlines

File: docs/get-started/preview_config.json
Comment (tgravescs) [inline docs/get-started/preview_config.json:1]: I'm probably missing it, is this referenced from somewhere in the docs?

Comment (sameerz) [reply docs/get-started/preview_config.json:1]: This is not needed, I will remove

**Comment by sameerz** [comment]
I think I resolved all the comments.  I am not sure how to resolve the conflicting files - I followed the command line instructions, but no changes appear to have been applied or resolved.

**Comment by jlowe** [comment]
>  I followed the command line instructions, but no changes appear to have been applied or resolved.

Did you have the latest branch-0.3 changes in your local repository when you performed the merge from branch-0.3?  That would explain the lack of changes.

**Comment by sameerz** [comment]
build

**Review by tgravescs — APPROVED** [review]

**Review by jlowe — APPROVED** [review]

**Pull request merged**
